Tasks/BashV3/bash.ts

```diff
@@ -122,9 +122,11 @@ async function run() {
 
         // Listen for stderr.
         let stderrFailure = false;
+        let aggregatedStderr = [];
```

Comment:
```
const aggregatedStderr:Buffer[] = []
```
Variable isnt reassigned so const is good. If you declare the array type you can get better assurances no one pushes the wrong thing to the array, and can also remove the type from the lambda when you read it because the type will be inferable by typescript
```
aggregatedStderr.forEach(err => tl.error(err.toString())
```